includes/digilan-db.php

```diff
@@ -172,8 +172,15 @@ public static function install_plugin_tables()
             dbDelta($sql);
         }
         add_option("digilan_token_version", $installed_version);
-        $query = "CREATE INDEX {$wpdb->prefix}digilan_token_index_mac ON {$wpdb->prefix}digilan_token_users_" . self::$installed_version . " (mac)";
-        $wpdb->query($query);
+
+        $sql = "SHOW INDEXES FROM `{$wpdb->prefix}digilan_token_users_" . self::$installed_version . "` WHERE `Key_name`='{$wpdb->prefix}digilan_token_index_mac'";
+        $result = $wpdb->query($sql);
+
+        if ($result === 0) {
+            $query = "CREATE INDEX {$wpdb->prefix}digilan_token_index_mac ON {$wpdb->prefix}digilan_token_users_" . self::$installed_version . " (mac)";
+            $wpdb->query($query);
+        }
+
```

Comment:
https://mariadb.com/kb/en/create-index/

MariaDB supporte ( OR REPLACE ) et IF NOT EXISTS

CREATE INDEX
Syntax
CREATE [OR REPLACE] [UNIQUE|FULLTEXT|SPATIAL] INDEX 
  [IF NOT EXISTS] index_name
    [index_type]
    ON tbl_name (index_col_name,...)
    [WAIT n | NOWAIT]
    [index_option]
    [algorithm_option | lock_option] ...

index_col_name:
    col_name [(length)] [ASC | DESC]